ee/dtr/admin/install/index.md

```diff
@@ -54,99 +58,68 @@ your profile on the left navigation pane, and select **Admin Settings > Docker T
 
-    For example, you could SSH into a UCP node and run the DTR install command
-    from there.  Running the installation command in interactive TTY or `-it` mode
-    means you will be prompted for any required additional information.  [Learn more about installing DTR](/reference/dtr/2.7/cli/install/).
+    * To install a specific version of DTR, replace `{{ page.dtr_version }}` with the  desired version in the [installation command](#step-3-install-dtr). (All DTR versions can be found on the [DTR release notes](/ee/dtr/release-notes/) page.)
 
-    Here are some useful options you can set during installation:
-    * To install a specific version of DTR, replace `{{ page.dtr_version }}` with
-    your desired version in the [installation command](#step-3-install-dtr)
-    above. Find all DTR versions in the [DTR release notes](/ee/dtr/release-notes/)
-    page.
+    * As DTR is deployed with self-signed certificates by default, UCP might not be able to pull images it. To automatically configure UCP to trust DTR, use the `--dtr-external-url <dtr-domain>:<port>` optional flag during installation, or during a reconfiguration.
 
-    * DTR is deployed with self-signed certificates by default, so UCP might not be
-    able to pull images from DTR. Use the `--dtr-external-url <dtr-domain>:<port>`
-    optional flag during installation, or during a reconfiguration, so that UCP is
-    automatically reconfigured to trust DTR.
+    * With DTR 2.7, it is possible to [enable browser authentication via client
+    certificates](/ee/enable-client-certificate-authentication/) at the time of install, and thus remove the need to enter user credentials.
 
-    * With DTR 2.7, you can [enable browser authentication via client
-    certificates](/ee/enable-client-certificate-authentication/) at install
-    time. This bypasses the DTR login page and hides the logout button, thereby
-    skipping the need for entering your username and password.
+5. Verify that DTR is installed, either by navigating to `https://<ucp-fqdn>/manage/settings/dtr`, or by navigating in the UCP web UI to **Admin Settings > Docker Trusted Registry**.
 
-4. Verify that DTR is installed. Either:
-    *  See `https://<ucp-fqdn>/manage/settings/dtr`, or;
+    Under the hood, UCP modifies `/etc/docker/certs.d` for each host and adds DTR's CA certificate. UCP can then pull images from DTR because the Docker Engine for each node in the UCP swarm has been configured to trust DTR.
```

Comment:
Suggest changing "under the hood" - could be confusing in languages other than English.

"...adds the DTR CA certificate." Avoid the use of trademarked terms as possessives.

Reply:
Excellent catches, Traci. Resolved.